helm/chart-operator-chart/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 description: A Helm chart for the chart-operator
 name: chart-operator-chart
-version: 0.3.2
+version: 0.4.0
```

Comment:
Bumps the version so we use a new release channel. This will be enabled in 0.12.0 of cluster-operator.